larq_compute_engine/tflite/python/interpreter_base.py

```diff
@@ -1,12 +1,13 @@
-from typing import Iterator, List, Tuple, Union, Optional
+from collections.abc import Iterator
+from typing import Union, Optional
 
 import numpy as np
 from tqdm import tqdm
 
-Data = Union[np.ndarray, List[np.ndarray]]
+Data = Union[np.ndarray, list[np.ndarray]]
 
 
-def data_generator(x: Union[Data, Iterator[Data]]) -> Iterator[List[np.ndarray]]:
+def data_generator(x: Union[Data, Iterator[Data]]) -> Iterator[list[np.ndarray]]:
```

Comment:
Reminder to ourselves that we should also upstream these changes to our internal repo.